### DISCUSSION TITLE
Is the temporary ban becoming a lot less temporary?

### DISCUSSION THREAD

**Discussion by evildrome** [kevinzg/facebook-scraper#854] [General]
The temporary ban used to be about 2 hours.

Now its 18 - 24hrs, for me anyway.  

Has anyone else had this issue?

**Comment by curiousier-george** [comment]
I haven't noticed this specifically myself (haven't gotten temporarily banned recently), but it seems to be consistent with my report and question #846.

**Reply by evildrome** [reply]
Yes.   I think I have also seen the problem you report in question #846
